tests/integration/commands/functions-with-args/functions-with-args.test.js

```diff
@@ -866,7 +866,7 @@ describe.concurrent('serving functions', () => {
           path: 'hello.js',
           handler: async () => ({
             statusCode: null,
-            body: 42,
+            body: 'hello',
```

Comment:
this was a test verifying the behaviour when `statusCode` is invalid, but it was also passing an invalid `body` (covered in another test)